content-src/static/activity-streams.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf8">
     <link rel="stylesheet" href="http://localhost:1963/main.css" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```

Comment:
We may be able to use a tool like https://icomoon.io/app/#/select/font to build a font-set with the 2-3 icons that we need instead of grabbing the full font family (even though I think the above file is only about 7.4KB minimized).

Ref: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome